aws/sdk/examples/ses/src/bin/email-contact-list.rs

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+use aws_types::region::ProvideRegion;
+
+use ses::model::{Body, Content, Destination, EmailContent, Message};
+use ses::{Client, Config, Error, Region};
+
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct Opt {
+    /// The contact list containing email addresses to send the message to.
+    #[structopt(short, long)]
+    contact_list: String,
+
+    /// The AWS Region.
+    #[structopt(short, long)]
+    default_region: Option<String>,
+
+    /// The email address of the sender.
+    #[structopt(short, long)]
+    from_address: String,
+
+    /// The message of the email.
+    #[structopt(short, long)]
+    message: String,
+
+    /// The subject of the email.
+    #[structopt(short, long)]
+    subject: String,
+    /// Whether to display additional runtime information
+    #[structopt(short, long)]
+    verbose: bool,
+}
+
+/// Sends a message to the email addresses in the contact list.
+/// # Arguments
+///
+/// * `-f FROM-ADDRESS` - The email address of the sender.
+/// * `-m MESSAGE` - The email message that is sent.
+/// * `-s SUBJECT` - The subject of the email message.
+/// * `-c CONTACT-LIST` - The contact list with the email addresses of the recepients.
+/// * `[-d DEFAULT-REGION]` - The region in which the client is created.
+///    If not supplied, uses the value of the **AWS_DEFAULT_REGION** environment variable.
+///    If the environment variable is not set, defaults to **us-west-2**.
+/// * `[-v]` - Whether to display additional information.
+#[tokio::main]
+async fn main() -> Result<(), Error> {
```

Comment:
It doesn't look like you're using the `?` operator anywhere, so `main` probably doesn't need to return a result.

Reply:
I would lean towards returning result & not using `expect()`. Better pattern for customers to avoid writing code that will ever panic.

Reply:
So no change?

Reply:
Yeah, I'm aligned with @rcoh.